packages/ckeditor5-dev-release-tools/lib/tasks/reassignnpmtags.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+/**
+ * Used to switch the tags from `staging` to `latest` for specified array of packages.
+ *
+ * @param {Object} options
+ * @param {String} options.npmOwner User that is authorized to release packages.
+ * @param {String} options.version Specifies the version of packages to reassign the tags for.
+ * @param {Array.<String>} options.packages Array of packages' names to reassign tags for.
+ * @returns {Promise}
+ */
+module.exports = async function reassignNpmTags( { npmOwner, version, packages } ) {
+	const errors = [];
+	const packagesSkipped = [];
+	const packagesUpdated = [];
+
+	await verifyLoggedInUserIsAuthorizedToPublish( npmOwner );
+
+	for ( const packageName of packages ) {
+		try {
+			const latestVersion = await exec( `npm show ${ packageName }@latest version` );
```

Comment:
This will throw when we publish a package for the first time.

Reply:
Fixed